src/test/datascience/ipywidgets/ipyWidgetScriptManager.vscode.common.test.ts

```diff
@@ -168,7 +168,7 @@ suite('IPyWidget Script Manager @widgets', function () {
         const newFiles = (await fs.getFiles(nbExtensionsFolder)).map((item) => item.toString());
         assert.deepEqual(newFiles, files);
     });
-    test('Get a list of Widgets and script paths', async () => {
+    test.skip('Get a list of Widgets and script paths', async () => {
```

Comment:
This will be fixed in a separate PR
tracked here  https://github.com/microsoft/vscode-jupyter/issues/14193